### PR TITLE
[author]update Day.js npm auto-update with latest 1.6.7 source code

### PR DESCRIPTION
**Pull request by iamkun**
Pull request for issue: #12828
Related issue(s): #12828

From dayjs@1.6.0, we changed the structure of our project.

We moved dayjs.min.js from `root/dist/dayjs.min.js` to` root/dayjs.min.js`.

Here is the pull request to update this changing.

Also update the repo url since `https://github.com/xx45/dayjs.git` will redirect to `https://github.com/iamkun/dayjs.git`

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Essential checklist
 * [x] I'm the author of this library

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/dayjs/package.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
...
-    "url": "https://github.com/xx45/dayjs.git"
+    "url": "https://github.com/iamkun/dayjs.git"
   },
   "npmName": "dayjs",
   "npmFileMap": [
     {
-      "basePath": "dist",
+      "basePath": "",
       "files": [
         "**/*"
```
Comment (sashberd) [inline ajax/libs/dayjs/package.json]: the pattern should be dayjs*

**Comment by iamkun** [comment]
Thanks @sashberd. Updated.

**Comment by sashberd** [comment]
@iamkun Squash?

**Comment by iamkun** [comment]
@sashberd Thanks, done pr #12834

**Pull request closed without merging**